Sources/Client/Client.h

```diff
@@ -527,6 +529,9 @@ namespace spades {
 			void LocalPlayerHurt(HurtType type, bool sourceGiven, Vector3 source) override;
 			void LocalPlayerBuildError(BuildFailureReason reason) override;
 			// IWorldListener end
+
+			static bool AreCheatsEnabled(); // 'cheats', i.e. spectator wallhack or player names
```

Comment:
As mentioned previously. Please change the names of the functions. They are not representative of what it is.

Its helpful for admins to check for players. So its not cheats.

WallhackActive -> SpectatorEspActive
AreCheatsEnabled -> AdminFeaturesEnabled